packages/dmn-js-decision-table/src/features/keyboard/Keyboard.js

```diff
@@ -206,7 +206,7 @@ export default class Keyboard {
       const selectedCell = cellSelection.getCellSelection();
 
       // add new rule if no next rule
-      if (!changed && selectedCell && !isDecisionTableIdCell(selectedCell)) {
+      if (!changed && selectedCell) {
```

Comment:
By removing this one we have to ensure the "add new rule on enter" action got not triggered on the name property input

![Kapture 2020-04-27 at 9 14 32](https://user-images.githubusercontent.com/9433996/80344226-86098000-8867-11ea-8c58-26b7cf9ebb8f.gif)

Reply:
Oh 😱 . I'm gonna fix that. Thanks!

Reply:
Fixed that.